.travis.yml

```diff
@@ -19,40 +20,29 @@ addons:
       - libeigen3-dev
 before_script:
   - export CXX="g++-4.8" CC="gcc-4.8" FC="gfortran-4.8"
-  - SUNBEAM_ROOT=(`pwd`)
+  - SUNBEAM_ROOT=$PWD
```

Comment:
The `SUN*` roots aren't really used. I don't mind keeping them for more verbose building & diagnostics, but it might signal an error.

Change at your discretion.